templates/taxi/manufacturer_confirm_delete.html

```diff
@@ -0,0 +1,13 @@
+{% extends "base.html" %}
+
+{% block content %}
+	<h1>Delete manufacturer</h1>
+
+  <p>Are you sure you want to delete the manufacturer: {{ manufacturer }}</p>
+  <p><i>All car with this manufacturer will be also deleted</i></p>
+  <form action="" method="post">

```

Comment:
The `action` attribute of the form is empty. Ensure that it points to the correct URL where the delete action should be processed. Typically, this would be the URL of the current page if the view is handling the form submission.